lib/al/Library/Camera/SnapShotCameraCtrl.h

```diff
@@ -3,48 +3,51 @@
 #include <al/Library/Audio/AudioKeeper.h>
 #include <al/Library/Collision/CollisionDirector.h>
 #include <al/Library/Nerve/NerveExecutor.h>
+#include <al/Library/Nerve/NerveSetupUtil.h>
 #include <al/Library/Yaml/ByamlIter.h>
-#include "gfx/seadCamera.h"
-
-struct CameraParam {
-    bool gotMin;
-    bool gotMax;
-    float mMinFovyDegree;
-    float mMaxFovyDegree;
-};
+#include <al/Library/Yaml/ByamlUtil.h>
+#include <gfx/seadCamera.h>
 
 namespace al {
 class SnapShotCameraSceneInfo;
 class ICameraInput;
 
+struct SnapShotCameraCtrlParam {  // Guessed name
```

Comment:
```suggestion
struct SnapShotParam { 
```
from `al::SnapShotCameraCtrl::load(al::ByamlIter const&)`